MapboxDirections/MBVisualInstructionComponent.swift

```diff
@@ -36,18 +36,22 @@ open class VisualInstructionComponent: NSObject, NSSecureCoding {
      :nodoc:
      `Bool` whether the `VisualInstructionComponent` should be visualized as a delimiter between two text components.
      */
-    @objc public var isDelimiter: Bool
+    @objc public var type: VisualCompomentType
     
     /**
      :nodoc:
      Initialize A `VisualInstructionComponent`.
      */
     @objc public convenience init(json: [String: Any]) {
         let text = json["text"] as? String
-        let isDelimiter = json["delimiter"] as? Bool ?? false
+        var type: VisualCompomentType?
+        if let delimiter = json["delimiter"] as? String, VisualCompomentType(description: delimiter) == .delimiter {
```

Comment:
In the response JSON, `delimiter` is either set to `true` or left undefined. It isn’t ever set to the string `delimiter`, as this code would suggest.